apphelpers/utilities/email.py

```diff
@@ -26,7 +26,7 @@ def send_email(
 
     msg["Subject"] = subject
     msg["From"] = sender
-    msg["To"] = recipient
+    msg["To"] = ', '.join(recipients)
```

Comment:
Check https://docs.python.org/3/library/smtplib.html#smtplib.SMTP.sendmail

> to_addrs: a **list** of RFC 822 to-address strings (a bare string will be treated as a list with 1 address)

So possibly we don't need string conversion here.
cc: @gauravr

Reply:
@shon List works at the time of calling `s.sendmail(sender, recipients, msg.as_string())`.
But for building msg, we need to pass string.